mappings/net/minecraft/datafixer/fix/LeavesFix.mapping

```diff
@@ -1,4 +1,22 @@
 CLASS net/minecraft/class_1191 net/minecraft/datafixer/fix/LeavesFix
+	FIELD field_5686 logBlocks Ljava/util/Set;
+	FIELD field_5688 leafMap Lit/unimi/dsi/fastutil/objects/Object2IntMap;
+	CLASS class_1192
```

Comment:
You probably should name this class.

Reply:
Honestly no clue what would be a proper name

Reply:
Looks like a `LeavesLogFixer` or such